services/pub/pub-downloads.service.js

```diff
@@ -0,0 +1,54 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+import { metric } from '../text-formatters.js'
+import { nonNegativeInteger } from '../validators.js'
+import { baseDescription } from './pub-common.js'
+
+const description = `${baseDescription}
+  <p>This badge shows a measure of how many developers have downloaded a package. This provides a raw measure of the overall sentiment of a package from peer developers.</p>`
+
+const schema = Joi.object({
+  downloadCount30Days: nonNegativeInteger,
+}).required()
+
+export default class PubDownloads extends BaseJsonService {
+  static category = 'downloads'
+
+  static route = { base: 'pub/downloads', pattern: ':packageName' }
+
+  static openApi = {
+    '/pub/downloads/{packageName}': {
```

Comment:
Some of our monthly downloads badges use `/dm` instead of `/downloads` in the route, using that instead would allow to extend the badge and have different periods covered in the future.

Reply:
:+1: 

see docs on this
https://github.com/badges/shields/blob/master/doc/badge-urls.md#:~:text=Downloads%20or%20Installs,day%3A%20/dd

Reply:
Updated as per docs.